feat(iOS 26, Stack v4): support for new search bar placements, integration with toolbar (#3168)
software-mansion/react-native-screens: guides/GUIDE_FOR_LIBRARY_AUTHORS.md

- `autoFocus` - If `true` automatically focuses search bar when screen is appearing. (Android only)
- `barTintColor` - The search field background color. By default bar tint color is translucent.
- `tintColor` - The color for the cursor caret and cancel button text. (iOS only)

> - `cancelButtonText` - The text to be used instead of default `Cancel` button text. (iOS only, **deprecated**)

Author: https://github.com/software-mansion/react-native-screens/pull/3168/commits/666afe244385cdf6e7e98c783f87ab7b4b05e466

> - `cancelButtonText` - The text to be used instead of default `Cancel` button text. (iOS only, **deprecated** starting from iOS 26)

Reviewer: ```suggestion
- `cancelButtonText` - The text to be used instead of default `Cancel` button text. *Has not effect as of iOS 26*. **Deprecated**. (iOS only)
```

Author: https://github.com/software-mansion/react-native-screens/pull/3168/commits/b43898fad323e8f97b516cbe26c42c49eb510763
